{{cookiecutter.project_slug}}/Procfile

```diff
@@ -14,6 +14,6 @@ web: gunicorn {{ cookiecutter.project_slug }}.wsgi --chdir=server --log-file -
 #   3. Up-to-date build numbers shown in app (ideally auto-generated at build-and-deploy time)
 #
 # Comment the line below to disable migrations automatically after a Heroku push.
-release: python server/manage.py makemigrations --noinput && python server/manage.py migrate --noinput && ./scripts/release.sh
+release: python server/manage.py makemigrations --noinput && python server/manage.py migrate --noinput
 
 
```

Comment:
What about test data? should that not get triggered?

Reply:
Test data gets run once during review app creation. That happens in a different place than this

Reply:
https://github.com/thinknimble/tn-spa-bootstrapper/blob/d2be249bccb8cd48584c56b29721dacb2417967f/%7B%7Bcookiecutter.project_slug%7D%7D/scripts/db_setup.sh#L6

`db_setup` is what triggers test data to be created.

Reply:
which is called from the `postdeploy` Heroku step.
https://github.com/thinknimble/tn-spa-bootstrapper/blob/4197058db8a941235135b58f051100b707edd238/%7B%7Bcookiecutter.project_slug%7D%7D/app.json#L47
That step only ever gets triggered during review app creation and thus never runs on staging or prod.
Which is why for brand new projects you have to run some of these commands manually once against staging.